core/src/main/scripts/importer/validateData.py

```diff
@@ -313,9 +317,14 @@ def _validate_file(self):
 
             # parse start-of-file comment lines, if any
             if not self.processTopLines(top_comments):
-                self.logger.error(
-                    'Invalid header comments, file cannot be parsed')
-                return
+                if not self.relaxed_mode:
+                    self.logger.error(
+                        'Invalid header comments, file cannot be parsed')
+                    return
+                else:   
+                    self.logger.info('Ignoring missing or invalid header comments. '
+                        'Continuing with validation...')                    
```

Comment:
I just noticed: this is something you should not need to change, it is more general than clinical data files. If we make a file without the header lines valid, `processTopLines` will not return `False` for those.